src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -27,7 +27,7 @@ public static void CompileDeepTree_NoStackOverflow(bool useInterpreter)
             Assert.Equal(n, f());
         }
 
-        [Theory, ActiveIssue("https://github.com/dotnet/corefx/issues/15533"), ClassData(typeof(CompilationTypes))]
+        [Theory, ClassData(typeof(CompilationTypes))]
```

Comment:
Passing 1 threw, passing 64K doesn't. I don't have a linux setup to test right now, but maybe there's a bug in how that is treated, or maybe the code in question just shouldn't be expected to work with a 16K stack.